packages/fxa-payments-server/src/store/index.tsx

```diff
@@ -69,134 +69,173 @@ export const selectors: Selectors = {
-          return { ...result, subscriptionId };
-        }),
-      reactivateSubscription: async (accessToken, subscriptionId) =>
-        apiPost(
-          accessToken,
-          `${config.servers.auth.url}/v1/oauth/subscriptions/reactivate`,
-          { subscriptionId }
-        ),
-      updatePayment: (accessToken, { paymentToken }) =>
-        apiPost(
-          accessToken,
-          `${config.servers.auth.url}/v1/oauth/subscriptions/updatePayment`,
-          { paymentToken }
-        ),
-    },
-    'updateApiData',
-    'resetCreateSubscription',
-    'resetCancelSubscription',
-    'resetReactivateSubscription',
-    'resetUpdatePayment'
-  ),
+export const actions = createActions(
+  {
+    fetchProfile: (accessToken: string) =>
+      apiGet(accessToken, `${config.servers.profile.url}/v1/profile`),
+    fetchPlans: (accessToken: string) =>
+      apiGet(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/plans`
+      ),
+    fetchSubscriptions: (accessToken: string) =>
+      apiGet(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/active`
+      ),
+    fetchToken: (accessToken: string) =>
+      apiPost(accessToken, `${config.servers.oauth.url}/v1/introspect`, {
+        token: accessToken,
+      }),
+    fetchCustomer: (accessToken: string) =>
+      apiGet(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/customer`
+      ),
+    createSubscription: (
+      accessToken: string,
+      params: {
+        paymentToken: string;
+        planId: string;
+        displayName: string;
+      }
+    ) =>
+      apiPost(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/active`,
+        params
+      ),
+    cancelSubscription: (accessToken: string, subscriptionId: string) =>
+      apiDelete(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/active/${subscriptionId}`
+      ).then(result => {
+        // HACK: cancellation response does not include subscriptionId, but we want it.
+        return { ...result, subscriptionId };
+      }),
+    reactivateSubscription: async (
+      accessToken: string,
+      subscriptionId: string
+    ) =>
+      apiPost(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/reactivate`,
+        { subscriptionId }
+      ),
+    updatePayment: (
+      accessToken: string,
+      { paymentToken }: { paymentToken: string }
+    ) =>
+      apiPost(
+        accessToken,
+        `${config.servers.auth.url}/v1/oauth/subscriptions/updatePayment`,
+        { paymentToken }
+      ),
+  },
+  'updateApiData',
+  'resetCreateSubscription',
+  'resetCancelSubscription',
+  'resetReactivateSubscription',
+  'resetUpdatePayment'
+);
 
-  // Convenience functions to produce action sequences via react-thunk functions
+// TODO: Find another way to handle these errors? Rejected promises result
```

Comment:
This is a weird bit. TL;DR: This swallows exceptions - which smells bad - but the errors *are* actually handled in the UI state. 

[The promise handling middleware both dispatches Redux actions for API errors *and* throws exceptions](https://github.com/pburtchaell/redux-promise-middleware/blob/master/docs/guides/rejected-promises.md). We handle the Redux actions, but the un-caught exceptions were bubbling up to the test runner. This squelches that.